build/toolchain/mac/BUILD.gn

```diff
@@ -276,10 +290,16 @@ mac_toolchain("clang_x64") {
 mac_toolchain("clang_x86") {
   toolchain_cpu = "i386"
   toolchain_os = "mac"
-  prefix = rebase_path(llvm_bin_path, root_build_dir)
-  ar = "$prefix/llvm-ar"
-  cc = "$prefix/clang"
-  cxx = "$prefix/clang++"
+  if (!enable_bitcode) {
```

Comment:
We can avoid this for host builds. `i386`

Reply:
Ditto